src/App/Fossa/VPS/Scan/Core.hs

```diff
@@ -2,33 +2,24 @@
 {-# LANGUAGE RecordWildCards #-}
 
 module App.Fossa.VPS.Scan.Core
-  ( createCoreProject
-  , completeCoreProject
-  , getSherlockInfo
-  , createLocator
+  ( createLocator
   , createRevisionLocator
-  , buildRevision
-  , overrideScanFilters
-  , storeUpdatedScanFilters
+  , getSherlockInfo
   , Locator(..)
   , RevisionLocator(..)
   , SherlockInfo(..)
   )
 where
 
 -- I REALLY don't like mixing these modules together.
```

Comment:
This comment is no longer relevant, which is just great.  Please take great pleasure in removing it.